### ISSUE TITLE
Delete row doesn't work

### ISSUE THREAD

**Issue by ckoehler** [moredip/Frank#3]
Hi again!

I tried to hit the delete button on a row using a simple:  
And I touch "Confirm Deletion for Foo"

That's the label that shows up when I inspect that button with the Accessibility Inspector, but it doesn't work, though that portion of the feature passes.

Not sure if it's relevant, but I found this post on the UISpec group:
http://groups.google.com/group/uispec/browse_thread/thread/1aba9e25d70bffce/ed446483e9208dd5?lnk=gst&q=delete#ed446483e9208dd5

Thanks!

**Comment by moredip** [comment]
This has been resolved by recent enhancements from Dale
